intf/intf.go

```diff
@@ -79,9 +85,12 @@ type NetworkAccessor interface {
 	// AddressAdd adds address ip to the interface name.
 	AddInterfaceIP(name string, ip *net.IPNet) error
 	// ARPList lists the set of ARP neighbours on the system.
-	ARPList() ([]*ARPNeigh, error)
+	ARPList() ([]*ARPEntry, error)
 	// ARPSubscribe writes changes to the ARP table to the channel updates, and uses
-	// done to indicate that the subscription is complete.
+	// done to indicate that the subscription is complete. The ARPSubscribe function
```

Comment:
for my understanding - I assume not-blocking because it is a "subscribe" function, and that naming implies non-blocking?

then if we had wanted to make a blocking version, would the naming be "ARPGet"?

Reply:
It could be blocking and mean that the caller has to put it into a goroutine if they want to not block, but it seemed more reasonable to just make it a non-blocking function itself (since this is the pattern that I expect most callers to use).